src/browser/components/molecules/Navbar/index.js

```diff
@@ -1,10 +1,12 @@
 import React, { Component } from 'react'
 import styles from './index.css'
+ import UserImage from '../../atoms/Userimage/userimage'
```

Comment:
if you rename `atoms/Userimage/userimage.js` to `atoms/Userimage/index.js` you can change this import statement to `import UserImage from '../../atoms/Userimage'` which looks much cleaner